src/constants.ts

```diff
@@ -12,7 +12,7 @@ import {
   decodePublicKey
 } from './utils'
 
-function readContextFile(ctxFile: string): ContextFile {
+export function readContextFile(ctxFile: string): ContextFile {

```

Comment:
please add documentation for all the exported function, interfaces, constants and enums